portal-api/src/v1/helpers/date.test.ts

```diff
@@ -88,36 +87,138 @@ describe('getStartOfDateFromDayMonthYearStrings', () => {
 
 describe('getStartOfDateFromDayMonthYearStringsReplicatingMoment', () => {
   const mockDay = '12';
-  const mockMonth = '05';
+  const mockMonth = '11';
   const mockYear = '2022';
-  it('returns the correct value when given valid dates', () => {
-    const result = getStartOfDateFromDayMonthYearStringsReplicatingMoment(mockDay, mockMonth, mockYear);
 
-    const expectedEpoch = 1652310000000; // Thu May 12 2022 00:00:00 GMT+0100
-    expect(result.valueOf()).toEqual(expectedEpoch);
-  });
+  describe('when currentDate is 8th March 2023', () => {
```

Comment:
can you make a random date a bit more middle of the year please (and call it `when currentDate is a non interesting day`

Reply:
have kept this as GMT day but added a BST date too